mycore-mods/src/main/java/org/mycore/mods/MCRMODSLinksEventHandler.java

```diff
@@ -91,8 +91,8 @@ protected void handleObjectUpdated(final MCREvent evt, final MCRObject obj) {
             MCRObjectID childID = childLinkID.getXLinkHrefID();
             if (MCRMetadataManager.exists(childID)) {
                 MCREvent childEvent
-                    = new MCREvent(MCREvent.ObjectType.fromClassName(childID.getTypeId()), MCREvent.EventType.INDEX);
```

Comment:
befor migration to objectType enum it was: `new MCREvent(childID.getTypeId(), MCREvent.TYPE_INDEX)`
which resolved to `mods` as first parameter - which would have been simply ignored?